.github/actions/generate-resources/README.md

```diff
@@ -35,7 +35,23 @@ node .github/actions/generate-resources/main.js --run;
 node .github/actions/generate-resources/node_modules/.bin/prettier README.md --write;
 ```
 
-## Example GitHub Action Usage 
+## Inputs
+
+None
+
+## Outputs
+
+None
+
+## Example usage 
+
+```yaml
+uses: actions/hello-world-javascript-action@v1.1
```

Comment:
Why are we doing an unrelated sample action?

Reply:
My bad, that should be reverted, copied the wrong action.